deepchecks/checks/performance/performance_report.py

```diff
@@ -77,3 +80,59 @@ def condition(result, min_score):
             return ConditionResult(True)
 
         return self.add_condition(name, condition, min_score=min_score)
+
+
+class MultiModelPerformanceReport(ModelComparisonBaseCheck):
+    """Summarize given scores between models on test datasets.
```

Comment:
```suggestion
    """Report score functions results for input models on test datasets.
```

Reply:
What's good about the word report? it weird-sounding as a verb